packages/board/src/vaadin-board-row.js

```diff
@@ -9,6 +9,7 @@
  * license.
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { isElementHidden } from '@vaadin/a11y-base/src/focus-utils.js';
```

Comment:
It looks like this needs to be added as a dependency? Board only has `components-base` ATM, which itself does not have `a11y-base` as dependency.

@web-padawan Can you confirm?

Reply:
Good catch, thanks. Yes, we need to add a dependency, otherwise there will be issues.

Reply:
Added the dependency.